src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1233,6 +1247,7 @@ def __call__(
         self,
         hidden_states,
         attention_mask,
+        indices_prng_key,
```

Comment:
We should remove the `indices_prng_key` argument from here onwards. It should not be passed as an arg to the self-attention layer. If you've got the rng dict approach correctly working, you should be able to remove all `indices_prng_key` args from here to the end of the modelling file!

```suggestion
```